app/models/ontology_version/parsing.rb

```diff
@@ -60,4 +61,21 @@ def parse_full
   def parse_fast
     parse(structure_only: true)
   end
+
+  def retrieve_available_provers_for_self_and_children
+    retrieve_available_provers
+    if ontology.distributed?
+      ontology.children.each do |child|
+        child.versions.find_by_commit_oid(commit_oid).retrieve_available_provers
+      end
+    end
+  end
+
+  def retrieve_available_provers
+    hets_options =
+      Hets::ProversOptions.new(:'url-catalog' => ontology.repository.url_maps,
+                               ontology: ontology)
+    provers_io = Hets.provers_via_api(ontology, hets_options)
+    Hets::Provers::Evaluator.new(self, provers_io).import
+  end
 end
```

Comment:
They are not explicitly tested, but implicitly by spec/models/ontology_version/provers_spec.rb